pkg/server/service.go

```diff
@@ -31,6 +33,8 @@ import (
 	imagesservice "github.com/containerd/containerd/services/images"
 	rootfsservice "github.com/containerd/containerd/services/rootfs"
 
+	"github.com/kubernetes-incubator/cri-o/pkg/ocicni"
```

Comment:
My bad, we should not have so many different groups of imports. Just add this to the second group. :)